src/remote.js

```diff
@@ -27,7 +27,8 @@
       usage: ['monaca remote build'],
       options: [
         ['--platform', 'Should be one of - ios, android, windows'],
-        ['--build-type', 'Should be one of - debug (for iOS, Android and Windows. It is default option.),'],
+        ['--build-type', 'Should be one of - debug (for iOS, Android and Windows. It is default option.)'],
+        ['--output', 'The path in which the built file will be stored (specify also the filename)'],
         ['', 'test (for iOS only),'],
```

Comment:
Which other options should be there, am I missing any one? There is a mistake here anyway: `--output` shold be after `release (for iOS, Android and Chrome Apps)`